cmd/scriggo/init_main.go

```diff
@@ -0,0 +1,84 @@
+package main
```

Comment:
It is ok to not include the copyright header in this file?

Reply:
Yes, this is the file in the generated interpreter.